### PR TITLE
LPS-183051 

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-183051

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/object/object-rest-test/src/testFunctional/tests/FilterParentCustomEntitiesWithDataFromRelatedElementsPt1.testcase [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+				en_US_plural_label = "students",
+				name = "Student1");
+
+			ObjectFieldAPI.updateFieldOfEntryById(
+				en_US_plural_label = "students",
+				entryId = ${student1Id},
+				fieldLabel = "description",
+				fieldValue = "8PLBPFx57V1IoHPEbWK4EpQPEwn3aAtW8mnvSuB6w2p4yiagg9cC8u5TJFh3b8q4k253zgK6y5DzP59xuHJlkg9nwFIsBEpAEiMdP84Hg8rZXnk7rthhwCJvq1wMhFDnAnPQw80rei7JK3nM8enCH0nj6GMLr8AvrjIrc5fb3tqn6M6YSE0IYR5HTU96G7Yfz9KligiuVdKk7TJVWdVIofY7DNrwnUuTtNRl4TP2emx051azT3Gz3T5k0vPOp3xr7LrlYcpu7oYlDG3nTAMQm6i6hMd30FMr4EvrA8YsN0F8jXzVXGG3Ro8ls910JEpY31TwlS76hlDQ9lmKD9Cei69WgfXx3MYlhRZ2B1bq5RnPtDbXw6hn7Tvjyo7LZ8L7x4xBrs6bke2Xi5V1sIvawCJsxKCK976aMe7vRZffDGFh59YXMXS4z6Fqp93C98jpqLlR9F6cU2k5G218rkikobMFBAUH7j",
+				requiredFieldLabel = "name",
+				requiredFieldValue = "Student2");
```
Comment (yanan) [inline modules/apps/object/object-rest-test/src/testFunctional/tests/FilterParentCustomEntitiesWithDataFromRelatedElementsPt1.testcase:138]: Shall we update the requiredFieldValue to Student1 here?

File: modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectFieldAPI.macro [hunk 2]
```diff
@@ -99,9 +99,13 @@ definition {
 					"${requiredFieldLabel}": "${requiredFieldValue}",
 		''';
 
-		if (isSet(pickList)) {
+		if (isSet(picklist)) {
 			var fieldInBody = '''"${fieldLabel}": {"key":"${fieldValue}"}''';
 		}
+
+		if (isSet(multiselectPicklist)) {
```
Comment (yanan) [inline modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectFieldAPI.macro:106]: :+1:

File: modules/apps/object/object-rest-test/src/testFunctional/tests/FilterParentCustomEntitiesWithDataFromRelatedElementsPt2.testcase [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+			HeadlessWebcontentAPI.assertProperNumberOfItems(
+				expectedTotalElement = 1,
+				responseToParse = ${response});
+		}
+	}
+
+	@disable-webdriver = "true"
+	@priority = 4
+	test CanFilterParentObjectEntriesInManyToManyRelByChildObjectLongTextAfterPublishingObject {
+		echo("Ready");
```
Comment (yanan) [inline modules/apps/object/object-rest-test/src/testFunctional/tests/FilterParentCustomEntitiesWithDataFromRelatedElementsPt2.testcase:309]: remove the echo for debug? :label:

**Review by yanan — COMMENTED** [review]
Others all good!

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**